Modules/Devices/test/Devices.Application.Tests/Tests/Devices/Commands/UpdateActiveDevice/HandlerTests.cs

```diff
@@ -30,75 +28,49 @@ public async Task Updates_device_with_values_sent()
         A.CallTo(() => fakeUserContext.GetDeviceId()).Returns(activeDevice.Id);
 
         var handler = CreateHandler(mockIdentitiesRepository, fakeUserContext);
-        var updateDeviceCommand = new UpdateDeviceCommand
+
+        var expectedCommunicationLanguage = CommunicationLanguage.Create("de");
+        var updateDeviceCommand = new UpdateActiveDeviceCommand
         {
-            CommunicationLanguage = "en"
+            CommunicationLanguage = expectedCommunicationLanguage.Value
         };
 
         // Act
         await handler.Handle(updateDeviceCommand, CancellationToken.None);
 
         // Assert
-        activeDevice.CommunicationLanguage.Should().Be(updateDeviceCommand.CommunicationLanguage);
+        activeDevice.CommunicationLanguage.Should().Be(expectedCommunicationLanguage.Value);
```

Comment:
Actually, I think that this assertion is redundant, as we already assert on what's passed to the repository.